partner_preview_scripts/encore_parser.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/env python3
+"""Parser for data submitted from the Encore team."""
+
+import argparse
+import json
+import logging
+import os
+import sys
+from functools import reduce
+
+
+from pyspark.sql import SparkSession
```

Comment:
```suggestion
from functools import reduce

from pyspark.sql import SparkSession
```

(Removing extra double blank line)